scripts/js/lib/api/updateLinks.ts

```diff
@@ -104,20 +108,10 @@ export function normalizeUrl(
     const normalizedParentName = kwargs.kebabCaseAndShorten
       ? kebabCaseAndShortenPage(parentName, kwargs.pkgName)
       : parentName;
-    return [...initialUrlParts, normalizedParentName].join("/") + "#" + member;
+    url = [...initialUrlParts, normalizedParentName].join("/") + "#" + member;
   }
 
-  if (!hash) return normalizedUrlWithoutHash;
-
-  // Anchors generated from markdown headings are always lower case but, if these
-  // headings are API references, Sphinx sometimes expects them to include
-  // uppercase characters.
```

Comment:
This came from the helper function `lowerCaseIfMarkdownAnchor`. I accidentally didn't delete it last PR. Its implementation was more robust so I use that rather than inlining